### ISSUE TITLE
bzminer and gminer can increase the memory frequency during ZIL mining plz...

### ISSUE THREAD

**Issue by yorocom119** [todxx/teamredminer#717]
bzminer and gminer can increase the memory frequency during ZIL mining, it's a pity that TRM can't do that.

**Comment by dibend** [comment]
I agree I would like to see TRM allow zil round to have altered memory clock so I can keep my power down dual mining
